modules/stream/dao_stream.c

```diff
@@ -335,6 +335,12 @@ static FILE* DaoIO_OpenFile( DaoProcess *proc, DString *name, const char *mode,
 	char buf[200];
 	FILE *fin;
 
+	if( Dao_IsDir( fname->chars ) ){
+		snprintf( buf, sizeof(buf), "is a directory: %s", DString_GetData( name ) );
```

Comment:
The description shall be "more descriptive" - something along the lines `fopen() undefined for directories: %s`.

Reply:
Added the more descriptive error